packages/app/src/specs/banners/RecordBanner.vue

```diff
@@ -16,47 +16,16 @@
     }"
     @update:model-value="value => emit('update:modelValue', value)"
   >
-    <p class="mb-24px">
```

Comment:
This copy should remain part of the banner. The RecordPrompt should just encapsulate the logic for creating the prompt itself.  That way, different places in the app that use it can have different copy surrounding the prompt.